lib/api/apiUtils/bucket/getReplicationConfiguration.js

```diff
@@ -0,0 +1,16 @@
+const parseXML = require('../../../utilities/parseXML');
+const ReplicationConfiguration = require('./models/ReplicationConfiguration');
+
+// Handle the steps for returning a valid replication configuration object.
+function getReplicationConfiguration(xml, log, cb) {
+    return parseXML(xml, log, (err, result) => {
+        if (err) {
+            return cb(err);
+        }
+        const validator = new ReplicationConfiguration(result, log);
+        const configErr = validator.buildConfigurationObject();
```

Comment:
where is buildConfigurationObject defined?  When I search I don't see it.  It's supposed to return an error?  Oddly named...